src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.test.tsx

```diff
@@ -122,11 +118,46 @@ it('renders non-donation table', async () => {
           total={0}
           currency={'USD'}
         />
-      </GqlMockedProvider>
-    </ThemeProvider>,
-  );
+      </ThemeProvider>,
+    );
 
-  expect(queryAllByRole('button')[0]).toBeInTheDocument();
+    expect(queryAllByRole('button')[0]).toBeInTheDocument();
+
+    expect(queryByTestId('donationColumn')).not.toBeInTheDocument();
+  });
+
+  it('links to contacts', async () => {
+    const data = gqlMock<
+      GetExpectedMonthlyTotalsQuery,
+      GetExpectedMonthlyTotalsQueryVariables
+    >(GetExpectedMonthlyTotalsDocument, {
+      variables: { accountListId: 'abc' },
+      mocks: {
+        expectedMonthlyTotalReport: {
+          likely: {
+            donations: [{ contactId: 'contact-1', contactName: 'John Doe' }],
+          },
+        },
+      },
+    });
 
-  expect(queryByTestId('donationColumn')).not.toBeInTheDocument();
+    const { findByRole } = render(
+      <ThemeProvider theme={theme}>
+        <ExpectedMonthlyTotalReportTable
+          accountListId={'abc'}
+          title={'Likely Partners This Month'}
+          data={data.expectedMonthlyTotalReport.likely.donations}
+          donations={false}
+          total={0}
+          currency={'USD'}
+        />
+      </ThemeProvider>,
+    );
```

Comment:
You could do this to remove some extra code:

```suggestion
  it('links to contacts', async () => {
    const { findByRole } = render(
      <ThemeProvider theme={theme}>
        <ExpectedMonthlyTotalReportTable
          accountListId={'abc'}
          title={'Likely Partners This Month'}
          data={[{ contactId: 'contact-1', contactName: 'John Doe' }]}
          donations={false}
          total={0}
          currency={'USD'}
        />
      </ThemeProvider>,
    );
```

Reply:
We need to pass in a whole donation object, but I realized you can mock the fragment instead of the whole query, which does simplify the mocking code. Thanks for pointing that out!